tools/admin/wskadmin

```diff
@@ -108,11 +108,11 @@ def parseArgs():
     subcmd = subparser.add_parser('whois', help='identify user from an authorization key')
     subcmd.add_argument('authkey', help='the credentials to look up')
 
-    subcmd = subparser.add_parser('block', help='block a user')
-    subcmd.add_argument('subject', help='the user to block')
+    subcmd = subparser.add_parser('block', help='block one or more users')
+    subcmd.add_argument('subject', help='comma separated list of users to block')
```

Comment:
Should this be

```python
subcmd.add_argument('subjects', nargs='+', help='comma separated list of users to block')
```

That would make the following command possible:

```
$ wskadmin user block subject1 subject2 subject3
```

which is more in line with bash standards I think. it also makes the "manual" splitting unnecessary (`args.subjects` will be a list already) and it'll force the user to give at least one subject.

WDYT?